packages/cli-link-assets/src/tools/manifest/index.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs-extra';
+import path from 'path';
+import migrations from './migrations';
+
+type Manifest = {
+  read: () => AssetPathAndSHA1[];
+  write: (data: AssetPathAndSHA1[]) => void;
+};
+
+type ManifestFile = {
+  migIndex: number;
+  data: AssetPathAndSHA1[];
+};
+
+type AssetPathAndSHA1 = {
+  path: string;
+  sha1: string;
+  shouldRelinkAndroidFonts?: boolean;
+};
+
+const migrationsLength = migrations.length;
+
+function readManifest(folderPath: string): ManifestFile {
+  return fs.readJsonSync(path.resolve(folderPath, 'link-assets-manifest.json'));
+}
+
+function writeManifest(folderPath: string, data: ManifestFile) {
+  return fs.writeJsonSync(
+    path.resolve(folderPath, 'link-assets-manifest.json'),
+    data,
+    {
+      spaces: 2,
+    },
+  );
+}
+
+const getManifest = (
+  folderPath: string,
+  platform: 'android' | 'ios',
```

Comment:
nit: I see that `'android' | 'ios'` few times over the changes, maybe we could move this into separate type?